plugins/env/test/samson_env/samson_plugin_test.rb

```diff
@@ -199,6 +201,17 @@ def render_view
         result = render_view
         result.wont_include checkbox
       end
+
+      it 'renders project erb link when DEPLOYMENT_ENV_REPO is present' do
+        with_env DEPLOYMENT_ENV_REPO: dep_env_repo do
+          result = render_view
+          result.must_include repo_link
+        end
+      end
+      it 'does not renders project erb link when DEPLOYMENT_ENV_REPO is nil' do
```

Comment:
newline between `it`s

Reply:
Done ... don't know why rubocop didn't pick it up?